cluster-autoscaler/cloudprovider/rancher/rancher_nodegroup.go

```diff
@@ -458,7 +458,7 @@ func parseResourceAnnotations(annotations map[string]string) (corev1.ResourceLis
 
 	cpuResources, err := resource.ParseQuantity(cpu)
 	if err != nil {
-		return nil, fmt.Errorf("unable to parse cpu resources: %s", cpu)
+		return nil, fmt.Errorf("unable to parse cpu resources: %w", err)
```

Comment:
Can we still include the string `cpu` in the error message in addition to `err`? That might also give a hint why the parsing has failed. Same goes for the other errors.

Reply:
Hi,

I'm not following..CPU is in the error for CPU?

Reply:
I mean the variable cpu a few lines above:

```go
cpu, ok := annotations[resourceCPUAnnotation]
```

If it can't be parsed by `resource.ParseQuantity(cpu)`, it would be helpful if the error message would contain the value. So the error would be something like:

```go
fmt.Errorf("unable to parse cpu resources from %q: %w", cpu, err)
```

Reply:
Ahh, yeah 👍🏻
I thought you meant the literal string 'cpu', which is what I removed previously for non-CPU values.

Wilo fix shortly.

Reply:
perfect, thanks!